src/main/java/seedu/address/logic/commands/AssignCommand.java

```diff
@@ -27,37 +26,33 @@
 public class AssignCommand extends Command {
     public static final String COMMAND_WORD = "assign";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a shift, worker and role assignment to the "
-            + "McScheduler by the index numbers used in the last worker and shift listings. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds shift, worker(s) and role(s) assignment(s) "
+            + "to the McScheduler by the index numbers used in the last worker and shift listings. "
             + "\nParameters: "
-            + PREFIX_SHIFT + "SHIFT_INDEX (must be a positive integer) "
-            + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer) "
-            + PREFIX_ROLE + "ROLE\n"
+            + PREFIX_SHIFT + "SHIFT_INDEX "
+            + PREFIX_WORKER_ROLE + "WORKER_INDEX ROLE\n"
```

Comment:
```suggestion
            + "{" + PREFIX_WORKER_ROLE + "WORKER_INDEX (must be a positive integer) ROLE}...\n"
```
`...` would show that the params can be repeated multiple times. I believe there is currently no convention to group a set of command prefixes and params with whitespaces between them together but perhaps we could use `{ }`.